crates/core/tedge/src/cli/certificate/cli.rs

```diff
@@ -186,3 +191,17 @@ pub enum UploadCertCli {
         profile: Option<ProfileName>,
     },
 }
+
+fn get_device_id_from_cloud(
```

Comment:
```suggestion
fn get_device_id_from_config(
```
Since it's not always the id from a cloud profile but could be the main `device.id` as well.